packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -55,9 +66,32 @@ export default function BookmarkFeedLayout({
       <FeedPageHeader className="mb-5">
         <h3 className="font-bold typo-callout">Bookmarks</h3>
       </FeedPageHeader>
-      <CustomFeedHeader className="relative mb-6">
+      <CustomFeedHeader className="flex relative mb-6">
         {searchChildren}
+
+        <Button
+          className="ml-4 btn-secondary hidden laptop:flex"
+          buttonSize="medium"
+          icon={<SourceIcon />}
+          onClick={() => setShowSharedBookmarks(true)}
+        >
+          Share bookmarks
+        </Button>
+        <Button
+          className="ml-4 btn-secondary block laptop:hidden"
+          buttonSize="medium"
+          icon={<SourceIcon />}
+          onClick={() => setShowSharedBookmarks(true)}
+          iconOnly
+        />
```

Comment:
Would love to hear from @sshanzel and @idoshamun what they think
I suggested this in favor of abusing the useMediaHook which becomes super heavy on the JS side of things.

Reply:
I am always in favor of utilizing CSS for any screen-related condition unless impossible.